api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java

```diff
@@ -66,14 +66,22 @@ public static WorkbenchException convertNotebookException(
     throw codeToException(e.getCode());
   }
 
-  public static WorkbenchException convertLeonardoException(
+  public static WorkbenchException convertLegacyLeonardoException(
```

Comment:
renamed many methods and classes related to the legacy Leo client, following up on #8838